ThunderCats/UITextField+TCAdditions.m

```diff
@@ -40,4 +40,27 @@ - (void)tc_setLeftAndRightPadding:(CGFloat)paddingWidth
     self.rightViewMode = UITextFieldViewModeUnlessEditing;
 }
 
+- (void)tc_addToolbarWithNextButton:(UITextField * __tc_nonnull)nextTextField
+                   toolbarColor:(UIColor * __tc_nonnull)toolbarColor
+{
+    [self tc_addToolbarWithButton:@"Next" toolbarColor:toolbarColor target:nextTextField action:@selector(becomeFirstResponder)];
+}
+
+- (void)tc_addToolbarWithButton:(NSString * __tc_nonnull)buttonTitle toolbarColor:(UIColor * __tc_nonnull)toolbarColor target:(id)target action:(SEL)selector
+{
+    UIBarButtonItem *button = [[UIBarButtonItem init] initWithTitle:buttonTitle style:UIBarButtonItemStylePlain target:target action:selector];
```

Comment:
Oh you're right, I'm still not sure how to test it. I tried using a breakpoint to see what the parameters are that are on the textfield but I can't figure it out.